indico/htdocs/sass/modules/_contributions.scss

```diff
@@ -21,3 +21,69 @@
 @import "contributions/layout";
 @import "contributions/display";
 @import "contributions/editor";
+
+
+@mixin item-picker-label() {
+    .label {
+        float: left;
+        max-width: 80%;
+        overflow: hidden !important;
+        text-overflow: ellipsis;
```

Comment:
that is wrapping of the root element (the element which you have to click to see the list of session), not the items inside the list.